components/menu-item/style.scss

```diff
@@ -27,6 +28,14 @@
 	&:focus:not(:disabled):not([aria-disabled="true"]) {
 		@include menu-style__focus;
 	}
+
+	// Forcefully colorize plugin icons to ensure contrast and cohesion.
+	svg,
+	g,
+	path {
+		stroke: $dark-gray-500 !important;
+		fill: $dark-gray-500 !important;
```

Comment:
Plugins can still override using `!important`. I'm not certain we should/can enforce that personally.

Reply:
The benefit of us doing this as strongly as we can, is that the plugin developer doesn't have to do anything on order to get the icon to work on a dark background.

While I don't feel super strongly here, I think we should _start_ out opinionated, and then we can consider pulling back if a valid use case presents itself.